### DISCUSSION TITLE
How to change datestyle in PostgreSQL Supabase

### DISCUSSION THREAD

**Discussion by joanfercho** [supabase/supabase#21201] [Questions]
Good afternoon, I want to know how to change the date format 'YYYY/MM/DD HH24:MI' to 'DD/MM/YYYY HH24:MI'

<img width="312" alt="Captura de pantalla 2024-02-12 155155" src="https://github.com/supabase/supabase/assets/31433528/ba2c9959-eb76-4b5b-a050-c154a0b7ea05">

**Comment by GaryAustin1** [comment]
The UI is going to display the date in Postgres's natural format.
If you want to map it in your API calls you will likely need a view or rpc call if you can't take care of it on the client.
https://github.com/PostgREST/postgrest/issues/177#issuecomment-88252296  Discusses using a view.
